docs/post-upgrade-req.asciidoc

```diff
@@ -1,22 +1,13 @@
 [[post-upgrade-req]]
 [role="xpack"]
-= Post upgrade requirements
+= Enable process analyzer after upgrade
 
-After upgrading the {stack} to version 7.9.x from a previous minor release
-(7.8.x, 7.7.x, and so on), you need to:
+To enable <<alerts-analyze-events, graphical representations of process relationships>>
+after upgrading to {stack} version 7.9.0 or 7.9.1 from a
```

Comment:
The fix is slated for 7.9.2 so this looks good 👍

Reply:
Worst case, I'll update the version numbers if needed.